Source/common/SantaCache.h

```diff
@@ -79,34 +70,33 @@ class SantaCache {
         (1 << (32 -
                __builtin_clz((((uint32_t)max_size_ / per_bucket) - 1) ?: 1)));
     if (unlikely(bucket_count_ > UINT32_MAX)) bucket_count_ = UINT32_MAX;
-    buckets_ = (struct bucket *)malloc(bucket_count_ * sizeof(struct bucket));
-    bzero(buckets_, bucket_count_ * sizeof(struct bucket));
+    buckets_ = new struct bucket[bucket_count_];
```

Comment:
How about making this a unique_ptr instead of dealing with new/delete?

Reply:
Done